juriscraper/oral_args/united_states/federal_appellate/cafc.py

```diff
@@ -6,54 +6,90 @@
-            date=d.strftime("%Y-%m-%d")
-        )
+        self.url = "https://cafc.uscourts.gov/wp-admin/admin-ajax.php?action=get_wdtable&table_id=8"
+        self.method = "POST"
+        self._set_parameters()
+        self.today = date.today().strftime("%m/%d/%Y")
+        self.earlier = (date.today() - timedelta(days=30)).strftime("%m/%d/%Y")
+        self.parameters[
+            "columns[0][search][value]"
+        ] = f"{self.earlier}|{self.today}"
         self.back_scrape_iterable = [
             i.date()
             for i in rrule(
-                DAILY,
-                interval=1,  # Every day
-                dtstart=date(2015, 7, 10),
-                until=date(2016, 4, 14),
+                MONTHLY,
+                dtstart=date(2021, 10, 8),
+                until=date(2022, 5, 17),
             )
         ]
 
-    def _get_download_urls(self):
-        path = "//td[contains(@class,'views-field-field-filename')]//@href"
-        return list(self.html.xpath(path))
+    def _set_parameters(self) -> None:
+        """Set the parameters for the request.
+
+        :return:None
+        """
+        self.parameters = {
+            "draw": "4",
+            "columns[0][data]": "0",
+            "columns[0][name]": "Arg_Date",
+            "columns[0][searchable]": "true",
+            "columns[0][orderable]": "true",
+            "columns[0][search][regex]": "false",
+            "columns[1][data]": "1",
+            "columns[1][name]": "Appeal_Number",
+            "columns[1][searchable]": "true",
+            "columns[1][orderable]": "true",
+            "columns[1][search][value]": "",
+            "columns[1][search][regex]": "false",
+            "columns[2][data]": "2",
+            "columns[2][name]": "Arg_Link",
+            "columns[2][searchable]": "true",
+            "columns[2][orderable]": "true",
+            "columns[2][search][value]": "",
+            "columns[2][search][regex]": "false",
+            "order[0][column]": "0",
+            "order[0][dir]": "desc",
+            "start": "0",
+            "length": "1000",
+            "search[value]": "",
+            "search[regex]": "false",
+            "wdtNonce": "6a2284f635",
+            "sRangeSeparator": "|",
+        }
 
-    def _get_case_names(self):
-        path = "//td[contains(@class,'views-field-title')]//text()"
-        return [" ".join(s.split()) for s in self.html.xpath(path)]
+    def _process_html(self) -> None:
+        """Extract content from JSON response
 
-    def _get_case_dates(self):
-        path = "//span[@class='date-display-single']/@content"
-        return [convert_date_string(s.strip()) for s in self.html.xpath(path)]
+        :return: None
+        """
+        for row in self.request["response"].json()["data"]:
+            _, name, _, url, _ = row[2].split('"')
+            self.cases.append(
+                {"date": row[0], "docket": row[1], "name": name, "url": url}
+            )
```

Comment:
A comment here with an example of the data would do wonders. Otherwise, LGTM, merge at will.

Reply:
Thanks.  I'll do that and send it off the PyPi in the sky.